word-search.py

```diff
@@ -237,4 +237,40 @@ def word_search():
     valid_words = load_word_library(grid)
   
 
+def continue_or_restart(): #function to enable the player to continue or quit in-between guesses
+    while True:
+        option = input("\nContinue Game? (y/n): ").strip().lower()
+        if option == 'y':
+            return True  #continue playing
+        elif option == 'n':
+            return False  #quit
+        else:
+            print("Invalid input. Please enter 'y' to continue or 'n' to restart.")
+
+def new_game():
+    while True: 
+        gridsize, timeroption = menu() #get grid size and timer option
+        word_search()  # start game
+
+
+        if timeroption == 4: #if untimed game is selected, ask the player if they want to continue or restart in between guesses
```

Comment:
based on word search process flow #47, user should be presented with game options 1. enter a word and 2. new game. code for entering word is missing and presenting options is missing.

Reply:
I think nasa line 252 na under menu(), well I could be wrong.

Reply:
grid and timer option po yung nasa 252. Ito po yata yung game menu pag unlimited time pero di ko din po sure if scope ng function niyo

![Screenshot 2024-11-30 215836](https://github.com/user-attachments/assets/d305bff2-9250-41c1-9f0c-899b8b5e4ffc)

Reply:
as per Vickie in group chat under ng task namin.

Reply:
medyo iba lang yung nagawa ko dyan check line 240 to 248 pero same essence >> option to continue or not after every guess pag untimed game